src/components/buttonCreator/ButtonCreator.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import "src/components/buttonCreator/ButtonCreator.component.scss";
+
+export const ButtonCreator = ({
```

Comment:
This would be much simpler and meaningful then ButtonCreator
```suggestion
export const Button = ({
```